pkg/controllerhelpers/error.go

```diff
@@ -19,3 +19,19 @@ func NewRequeueError(reasons ...string) *RequeueError {
 func (e *RequeueError) Error() string {
 	return strings.Join(e.reasons, ", ")
 }
+
+type PermanentError struct {
```

Comment:
It's not used anywhere

Reply:
thx, removed the leftover from previous iterations